features/FxA-56-email-deliverability/README.md

```diff
@@ -1,66 +1,160 @@
-# Email deliverability (FxA-56)
+# Email Deliverability (FxA-56)
 
 Feature card: https://github.com/mozilla/fxa-features/issues/56
 
 ## Problem Summary
 
-The Firefox Accounts email deliverability is suffering from
-lack of understanding of the common causes of email bounces. There are not enough metrics to detect email bounce reasons and understand the cause of issues.
+Users regularly report email deliverability issues with Firefox Accounts,
+and we do not have sufficient visibility into
+the common causes of email bounces and complaints
+to detect or debug them.
 
 ----
 
 ## Outcomes
 
-To have better metrics and logging around our email delivery, with
-the goal of improving email deliverability afterwards.
+To have better metrics and logging around our email delivery,
+so we can judge the effectiveness of any attempts to improve
+our delivery rate.
+
+To try one simple tactic for improving email deliverability
+and measure whether it is successful.
 
 ## Hypothesis
 
-If we add metrics to email services in FxA then
-**we will have a chance to improve the email deliverability** of the
-authentication system.
+If we add email deliverability metrics to FxA then
+**we can take a data-driven approach to improving email deliverability**
+of the authentication system.
 
-If we stop sending additional email messages to bounced emails addresses then our "complaint rate" with popular email providers will decrease and improve our deliverability.
+If we avoid sending additional emails to addresses that are known to bounce then
+**our "sender reputation" wth popular email providers will improve**
+and hence our email deliverability will improve.
 
 
 ## Metrics
 
-The following metrics should help us understand and debug email deliverability:
+The following metrics will be used
+to understand and debug email deliverability:
 
-* Flow events for email bounces
-* Current bounce rate
-* A dashboard that lists common
+### Completion Rate for Email Flows
 
-----
+For flows that result in an email being sent,
+we will produce a flow funnel graph
+showing the rate of dropoff for the following events:
 
-## Detailed design
+* The email being successfully delivered by SES
+* The link in the email being clicked
+* The flow completing
+
+The graph will look something like this:
+
+![Mock graph for completion rate of email flows](email_flow_completion_rate.jpg)
```

Comment:
This looks great. We can add these flow events to our main funnel too.

There are just 2 things missing that I can think of: open rate and click rate by email provider.

But, we don't really need to measure open rate for now since click rate (unique clicks / delivered) is a great indicator of if we are still landing in the inbox or if we have started to land in spam (and is more accurate since open rate can be dependent on email provider).

Probably the best way to plot click rate is by email provider (gmail, outlook, etc.). This is super important because if we have a 30% clickrate with Gmail but only 3% with Outlook, it is a strong a sign that we are landing in spam with Outlook.